styx-api-service/src/test/java/com/spotify/styx/api/workflow/WorkflowInitializerTest.java

```diff
@@ -63,7 +63,7 @@
   @Mock private StorageTransaction transaction;
 
   @Before
-  public void setUp() throws IOException {
+  public void setUp() throws Exception {
```

Comment:
What caused this change?

Reply:
Javac bug in JDK 11 

https://github.com/danielnorberg/jdk11-behavior-change-repro

Reply:
Awesome!